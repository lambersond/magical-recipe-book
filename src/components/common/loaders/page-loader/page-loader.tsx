export function PageLoader() {
  return (
    <div className='fixed flex items-center justify-center h-screen w-screen inset-0'>
      <div className='loader border border-secondary w-[25vw]' />
      <p className='absolute text-center text-sm lg:text-lg font-semibold text-text-secondary overflow-visible'>
        Loading...
      </p>
    </div>
  )
}
