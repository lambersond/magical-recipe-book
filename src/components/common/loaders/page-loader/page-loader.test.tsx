import { render } from '@test-utils'
import { PageLoader } from './page-loader'

describe('components/common/loaders/page-loader', () => {
  it('should match snapshot', () => {
    expect(render(<PageLoader />).asFragment()).toMatchSnapshot()
  })
})
