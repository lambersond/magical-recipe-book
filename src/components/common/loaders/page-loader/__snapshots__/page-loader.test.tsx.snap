// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/common/loaders/page-loader should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="fixed flex items-center justify-center h-screen w-screen inset-0"
  >
    <div
      class="loader border border-secondary w-[25vw]"
    />
    <p
      class="absolute text-center text-sm lg:text-lg font-semibold text-text-secondary overflow-visible"
    >
      Loading...
    </p>
  </div>
</DocumentFragment>
`;
